app/models/client_opportunity_match.rb

```diff
@@ -658,6 +658,9 @@ def restrictions_on_reopening
   def reopen!(contact, user: nil)
     self.class.transaction do
       client.make_unavailable_in(match_route: match_route, expires_at: nil, user: user, match: self, reason: UnavailableAsCandidateFor::ACTIVE_MATCH_TEXT)
+      match_route.routes_parked_on_active_match.reject(&:empty?).each do |park_route|
```

Comment:
```suggestion
      # Park client on any other routes where this route is set to block matching
      match_route.routes_parked_on_active_match.reject(&:empty?).each do |park_route|
```

Reply:
This has been included in the lates commit